pkg/glusterfs/controllerserver.go

```diff
@@ -0,0 +1,377 @@
+package glusterfs
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	csi "github.com/container-storage-interface/spec/lib/go/csi/v0"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/golang/glog"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+)
+
+const (
+	glusterDescAnn        = "GlusterFS-CSI"
+	glusterDescAnnValue   = "gluster.org/glusterfs-csi"
+	defaultVolumeSizeInMB = 20 * MB // minimum volume size
+)
+
+const (
+	_  = iota
+	KB = 1 << (10 * iota)
+	MB
+	GB
+	TB
```

Comment:
If we have to support human friendly values, I'd prefer we use a well used package instead of rolling our own. The go-humanize package is used widely, and I believe K8S uses it as well to support human friendly values in the definition files.